src/oas.js

```diff
@@ -101,6 +102,80 @@ function normalizedUrl(oas) {
   return ensureProtocol(url);
 }
 
+function normalizePath(path) {
+  const curlBacketMatch = /{(.*?)}/;
+
+  return path
+    .split('/')
+    .map(p => {
+      const pathMatch = curlBacketMatch.exec(p);
+      if (pathMatch) return `:${pathMatch[1]}`;
+      return p;
+    })
+    .join('/');
+}
+
+function generatePathMatches(paths, pathName, origin) {
+  return Object.keys(paths)
```

Comment:
What happens when there are multiple matches here? Can there be multiple matches in this step?

Reply:
" it deals with multiple matches by prioritizing the match with the least path Parmas. Those wildcards really are the ones that would contribute to finding dups 
If there are other dups (like repeat paths) that would just be an incorrect oas file to begin with" via our slack